libs/vertexai/langchain_google_vertexai/_base.py

```diff
@@ -189,6 +196,7 @@ def _default_params(self) -> Dict[str, Any]:
             "temperature": self.temperature,
             "max_output_tokens": self.max_output_tokens,
             "candidate_count": self.n,
+            "response_mime_type": self.response_mime_type,
```

Comment:
I don't think it should be there, since `response_mime_type` is not support for embeddings.

Reply:
It should be rather a redefined property on a chat model that uses super()._default_params

Reply:
removed